external-docs/docs/guides/configuration/templates.md

```diff
@@ -80,3 +80,89 @@ Units used, have to match the terminal part of the URI exactly including case se
+
+| Period     | Frequency | ... |
+|------------|-----------|-----|
+| 2019-04-01 | Daily     | ... |
+| 2019-04-02 | Daily     | ... |
+| 2019-04-01 | Weekly    | ... |
+| 2019-04-08 | Weekly    | ... |
+
+**NB:** it is important to validate whether the weekly data matches the ISO8601 definition, as this is slightly different from the traditional Gregorian calendar.
+
+**Things to note:**
+
+- The week date representation must commence on a Monday. If not, a [gregorian-interval](https://purl.org/csv-cubed/qube-config/templates/gregorian-interval.json) of one week must be used.
+- Unlike the Gregorian calendar, there are no weeks that extend across years. Each ISO8601 year consists of either 52 or 53 weeks depending on when the year begins. If 1 January falls on a Monday, Tuesday, Wednesday or Thursday it is in week 01 of the new year. If 1 January falls on a Friday, Saturday or Sunday, it is in week 52 or 53 of the previous year.
+- More information can be found [here](https://en.wikipedia.org/wiki/ISO_8601#Week_dates)
+
+### How to format mixed periods
+
+- For daily data, prefix the date with `day/`: `day/YYYY-MM-DD`
+- For weekly data, calculate the ISO8601 week value and prefix with `week/`: `week/YYYY-WWW`. For example, the week commencing 01/04/2019 would be `week/2019-W14`.
+- Once the dates have been formatted as above, the 'Frequency' column should be removed.
+
+### Example Python code
+
+The following code is an example of how this can be achieved in Python:
+
+```python
+import click
+import pandas as pd
+import numpy as np
+from pathlib import Path
+
+@click.command()
+@click.argument("input", type=click.Path(exists=True, path_type=Path))
+@click.option("--output", default=Path("./output.csv"), type=click.Path(path_type=Path))
+def wrangle(input: Path(), output: Path()) -> None:
+    df = pd.read_csv(input_file_path)
+
+    # Convert dates which represent weeks to correct format 'YYYY-WWW'
+    df['Period'].loc[df['Frequency']] = np.where(df['Frequency'] == 'Weekly', pd.to_datetime(df['Period']).dt.strftime('%Y-%V'), df['Period'])
+
+    # Prefix the date with `week/` when Weekly is used in frequency column
+    df['Period'] = np.where(df['Frequency'] == 'Weekly', ('week/' + df['Period']), df['Period'])
+
+    # Prefix the date with `day/` when Daily is used in frequency column
+    df['Period'] = np.where(df['Frequency'] == 'Daily', ('day/' + df['Period']), df['Period'])
+
+    # Drop 'Frequency' column
+    df.drop("Frequency", axis=1, inplace=True)
+
+    # Save the updated file as CSV
+    df.to_csv(output_file_path, index=False)
+    return
+
+if __name__ == "__main__":
+    wrangle()
+```
+
+
```

Comment:
Should we give the user the command to run this. Again, I know it seems obvious to us how to run this, but it might not seem obvious to someone else.